plugin/trino-hive/src/main/java/io/trino/plugin/hive/parquet/ParquetPageSourceFactory.java

```diff
@@ -384,6 +429,14 @@ public static Optional<ColumnIndexStore> getColumnIndexStore(
         return Optional.of(new TrinoColumnIndexStore(dataSource, blockMetadata, columnsReadPaths, columnsFilteredPaths));
     }
 
+    public static Optional<BloomFilterStore> getBloomFilterStore(ParquetDataSource dataSource, ParquetReaderOptions options)
```

Comment:
This can be a static method in BloomFilterStore

Reply:
hi, since the method starting from line 350 is called "getColumnIndexStore" I followed the convention. Since it is a static method, it can certainly be in other classes. Do you still need to move this class?